api/http/api_useradm.go

```diff
@@ -136,6 +140,38 @@ func (u *UserAdmApiHandlers) AuthVerifyHandler(w rest.ResponseWriter, r *rest.Re
 	w.WriteHeader(http.StatusOK)
 }
 
+func (u *UserAdmApiHandlers) CreateTenantUserHandler(w rest.ResponseWriter, r *rest.Request) {
+	ctx := r.Context()
+
+	l := log.FromContext(ctx)
+
+	user, err := parseUser(r)
+	if err != nil {
+		if err == model.ErrPasswordTooShort {
+			rest_utils.RestErrWithLog(w, r, l, err, http.StatusUnprocessableEntity)
+		} else {
+			rest_utils.RestErrWithLog(w, r, l, err, http.StatusBadRequest)
+		}
+		return
+	}
+
+	ctx = getTenantContext(ctx, r.PathParam("id"))
+
+	err = u.userAdm.CreateUser(ctx, &user.User, user.Propagate)
+	if err != nil {
+		if err == store.ErrDuplicateEmail {
+			rest_utils.RestErrWithLog(w, r, l, err, http.StatusUnprocessableEntity)
+		} else {
+			rest_utils.RestErrWithLogInternal(w, r, l, err)
+		}
+		return
+	}
+
+	w.Header().Add("Location", "users/"+string(user.ID))
```

Comment:
We currently do not have an endpoint `GET /api/internal/v1/useradm/tenants/:id/users/:uid`, so I think it's ok to skip the `Location` header here.